python/glow/gwas/lin_reg.py

```diff
@@ -32,30 +32,54 @@ def linear_regression(genotype_df: DataFrame,
     In each Spark task, we project the covariates out of a block of genotypes and then compute the regression statistics for each phenotype,
     taking into account the distinct missingness patterns of each phenotype.
 
```

Comment:
Can we add a less technical version of this paragraph to the Linear Regression doc page to echo the power of implementation?